src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_RMSMappingQuality.java

```diff
@@ -58,26 +59,51 @@
     public Map<String, Object> annotate(final ReferenceContext ref,
                                         final VariantContext vc,
                                         final ReadLikelihoods<Allele> likelihoods) {
-        return annotateRawData(ref, vc, likelihoods);
+        Utils.nonNull(vc);
+        if ( likelihoods == null) {
+            return Collections.emptyMap();
+        }
+        final Map<String, Object> annotations = new HashMap<>();
+        final ReducibleAnnotationData<Number> myData = new ReducibleAnnotationData<>(null);
+        getRMSDataFromLikelihoods(likelihoods, myData);;
+        final String annotationString = makeFinalizedAnnotationString(vc, myData.getAttributeMap());
+        annotations.put(getKeyNames().get(0), annotationString);
+        return annotations;
     }
 
+
     @Override
     public Map<String, Object> annotateRawData(final ReferenceContext ref,
                                                final VariantContext vc,
                                                final ReadLikelihoods<Allele> likelihoods ) {
         Utils.nonNull(vc);
-        if ( likelihoods == null) {
+        if ( likelihoods == null || !likelihoods.hasFilledLiklihoods()) {
             return Collections.emptyMap();
         }
 
         final Map<String, Object> annotations = new LinkedHashMap<>();
         final ReducibleAnnotationData<Number> myData = new ReducibleAnnotationData<>(null);
-        calculateRawData(vc, likelihoods, myData);
+        getRMSDataFromLikelihoods(likelihoods, myData);;
```

Comment:
double semicolon here too -- oh, the dangers of copy and paste.

Reply:
done